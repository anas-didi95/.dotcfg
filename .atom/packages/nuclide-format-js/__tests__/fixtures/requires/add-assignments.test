var one = two;
three = four;
var five = six, seven;
var {eight} = nine;
