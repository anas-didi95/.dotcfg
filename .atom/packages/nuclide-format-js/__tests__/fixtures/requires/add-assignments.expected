const four = require('four');
const nine = require('nine');
const six = require('six');
const three = require('three');
const two = require('two');

var one = two;
three = four;
var five = six, seven;
var {eight} = nine;
